iree/base/internal/BUILD

```
@@ -105,53 +102,25 @@ cc_library(
     visibility = ["//visibility:private"],
 )
 
-cc_library(
-    name = "source_location_internal",
-    hdrs = ["source_location.h"],
-)
-
 cc_library(
     name = "status_internal",
     srcs = [
         "status.cc",
         "status_builder.cc",
-        "status_errno.cc",
-        "status_errors.cc",
-        "status_win32_errors.cc",
         "statusor.cc",
     ],
     hdrs = [
         "status.h",
         "status_builder.h",
-        "status_errno.h",
-        "status_errors.h",
-        "status_macros.h",
-        "status_win32_errors.h",
         "statusor.h",
     ],
     deps = [
         ":logging_internal",
```

Please use base:logging now that this target is unconditionally included in the build
```suggestion
        "//iree/base:logging",
```